<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>标签1</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0" rowspan="3">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>GroupBox</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QPushButton" name="btn_1">
             <property name="text">
              <string>修改状态栏</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="btn_2">
             <property name="text">
              <string>显示图片</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="btn_3">
             <property name="text">
              <string>开启线程</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="btn_4">
             <property name="text">
              <string>停止线程</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="btn_5">
             <property name="text">
              <string>暂停线程</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="btn_6">
             <property name="text">
              <string>启动线程</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QPushButton" name="btn_7">
             <property name="text">
              <string>添加列表</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="btn_8">
             <property name="text">
              <string>计时</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QPushButton" name="btn_9">
             <property name="text">
              <string>选择文件</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QPushButton" name="btn_10">
             <property name="text">
              <string>选择文件夹</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QPushButton" name="btn_11">
             <property name="text">
              <string>保存文件</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QPushButton" name="pushButton_6">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QPushButton" name="pushButton_5">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QPushButton" name="pushButton_8">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QPushButton" name="pushButton_14">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QPushButton" name="pushButton_11">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QPushButton" name="pushButton_9">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QPushButton" name="pushButton_15">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QPushButton" name="pushButton_16">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QPushButton" name="pushButton_12">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QPushButton" name="pushButton_10">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item row="10" column="1">
            <widget class="QPushButton" name="pushButton_13">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item row="11" column="0">
            <widget class="QPushButton" name="pushButton_23">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item row="11" column="1">
            <widget class="QPushButton" name="pushButton_22">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item row="12" column="0">
            <widget class="QPushButton" name="pushButton_24">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item row="12" column="1">
            <widget class="QPushButton" name="pushButton_19">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item row="13" column="0">
            <widget class="QPushButton" name="pushButton_20">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item row="13" column="1">
            <widget class="QPushButton" name="pushButton_18">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>label框</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="pic_label">
             <property name="minimumSize">
              <size>
               <width>361</width>
               <height>221</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>2000</width>
               <height>2000</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="2" rowspan="3">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="minimumSize">
           <size>
            <width>331</width>
            <height>433</height>
           </size>
          </property>
          <property name="title">
           <string>GroupBox</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QTableWidget" name="tableWidget">
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>150</number>
             </attribute>
             <column>
              <property name="text">
               <string>操作</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>时间</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>TextEdit框</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="0">
            <widget class="QTextEdit" name="textEdit">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="html">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QListWidget" name="listWidget"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>标签2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>942</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
